setup.py

```diff
@@ -887,7 +887,9 @@ def build_extensions(self):
         else:
             self._remove_extension("PIL._webp")
 
-        tk_libs = ["psapi"] if sys.platform == "win32" else []
+        tk_libs = (
+            ["psapi"] if (sys.platform == "win32" or sys.platform == "cygwin") else []
+        )
```

Comment:
```suggestion
tk_libs = ["psapi"] if sys.platform in ("win32", "cygwin") else []
```
Minor formatting suggestion.

Reply:
Done